src/simulation_counties.R

```diff
@@ -5,25 +5,23 @@ library(parallel)
 source("hmm_functions.R")
 
 
-simulation_df <- data.frame(n_points_per_county=c(100, 500, rep(1000,13)),
-                            n_counties=c(rep(100,15)),
-                            n_time_periods=c( rep(4, 10),5,6,rep(4,3)),
-                            n_components = c(rep(2, 15)),
-                            mu1 = c(rep(90,15)), ##Put in as percent
-                            defRt1 = c(rep(4,15)), ##In as percent
-                            defRtMid = c(rep(10,15)), ##same
-                            defRtLast = c(20, 20, seq(5,40,by=5),rep(20,5)),
-                            prY11 = c(rep(90,12),75,80,95), ##Correct classification Probability 1,1 in matrix
-                            prY22 = c(rep(80,15))) ##Correct classification Probability 2,2 in matrix
+## TODO "n_counties" should really be "n_replications", since we're no longer running county regressions
```

Comment:
@rosenbat2 does this TODO make sense to you?  Eduardo had originally discussed a simulation in which we had variation in fixed effects and county-level predictors (and therefore in deforestation rates) across counties, and ran regressions to recover the effect of county-level variables on deforestation rates (using naive, EM and MD estimates of deforestation rates).  We've moved away from that and toward a simulation of sampling distributions (the "counties" are identical, they're really just i.i.d. simulation replications), so the "county" language doesn't really make sense anymore.  Just want to confirm before I change the variable names.